src/brsTypes/BrsType.ts

```diff
@@ -389,7 +389,7 @@ export class Uninitialized implements BrsValue, Comparable {
 
     equalTo(other: BrsType): BrsBoolean {
         if (other.kind === ValueKind.String) {
-            // Allow variables to be compared to the string "<UNINITIALIZED>" to test if they've
+            // Allow variables to be compared to the string "<uninitialized>" to test if they've
             // been initialized
             return BrsBoolean.from(other.value === this.toString());
```

Comment:
It looks like `print someVariableThatDoesntExist` will print capitalized `<UNINITIALIZED>`, but `print type(someVariableThatDoesntExist)` will print lowercase `<uninitialized>`.  Similarly, `print doesNotExist = "<UNINITIALIZED>"` is `false`, but `print doesNotExist = "<uninitialized>"` is `true`.  We'll probably need to make this compare against `.toString().toLowerCase()` and revert the `toString()` change below!

```suggestion
            return BrsBoolean.from(other.value === this.toString().toLowerCase());
```